src/containers/Main/Main.test.tsx

```diff
@@ -210,4 +210,33 @@ describe("Main", () => {
       }
     });
   });
+  it.only("filters network data with the given search query inverted", async () => {
```

Comment:
Please remove .only

Reply:
nice catch thanks, I was just testing it and didn't notice that left it like this.

Reply:
```suggestion
  it("filters network data with the given search query inverted", async () => {
```